solidity/depool/README.md

```diff
@@ -318,6 +318,10 @@ TONOS-CLI allows to manage several types of stakes. For details on stake types r
 
 Below are listed the commands used to manage stakes.
 
+> Note: All these commands are subject to an additional fee (by default 0.5 tons), that is partially spent to pay for DePool executing the command. The change is then returned to the sender. This value can be adjusted in TONOS-CLI config.
+
+> Note: Additionally, when DePool receives the stake and rewards back from elector and processes the funds of participants, 0.05 tons are deducted from every participant's share of the pool, to cover the costs of executing this action.
```

Comment:
It might be more usefull to mention that this 0.05 tons is RET_OR_REINV_FEE and it coulfd be obtained via calling getDePoolInfo

Reply:
added